tests/testthat/test-subset-access.R

```diff
@@ -206,6 +220,21 @@ test_that("top_left retention behavior is correct across all scenarios", {
     expect_identical(top_left(tbl[1:2, 1:2]), character())
     expect_identical(top_left(tbl[1:2, 1:2, keep_topleft = FALSE]), character())
     expect_identical(top_left(tbl[1:2, 1:2, keep_topleft = TRUE]), tlval)
+    
+    # error for multiple elemented selected for dropping
+    expect_error(tbl[1:2, 1, drop = TRUE])
```

Comment:
Not error, see above comment

Reply:
I do not see the comment. For changing this behavior, I created an issue in https://github.com/Roche/rtables/issues/475